server.py

```diff
@@ -171,6 +172,21 @@ def testpostgres():
             conn.close()
 
 
+# Search service
+@app.route(PATH_SEARCH)
+@app.route(PATH_SEARCH + '<string:tenant_id>/')
+def search_url_get(tenant_id=None):
+    response_code = request.args.get(RESPONSECODE, default=200, type=int)
+    return json.dumps({RESPONSECODE: response_code, "tenantID": tenant_id}), response_code
+
+
+@app.route(PATH_SEARCH + '<string:tenant_id>/model', methods=['POST'])
+@app.route(PATH_SEARCH + '<string:tenant_id>/index', methods=['POST'])
+@app.route(PATH_SEARCH + '<string:tenant_id>/query', methods=['POST'])
+def search_url_post(tenant_id):
+    return json.dumps({"Status": "Ok", "tenantID": tenant_id}), 200
```

Comment:
For testing purposes, it would be good to include the action (index/query/model) as part of the response